src/odemis/driver/test/xt_client_test.py

```diff
@@ -821,10 +823,10 @@ def test_beamlet_index_info(self):
         beamlet_index_info = self.microscope.beamlet_index_info()
         self.assertIsInstance(beamlet_index_info, dict)
         self.assertIsInstance(beamlet_index_info["range"], dict)
-        self.assertIsInstance(beamlet_index_info["range"]["x-direction"], list)
-        self.assertIsInstance(beamlet_index_info["range"]["y-direction"], list)
-        self.assertEqual(len(beamlet_index_info["range"]["x-direction"]), 2)
-        self.assertEqual(len(beamlet_index_info["range"]["y-direction"]), 2)
+        self.assertIsInstance(beamlet_index_info["range"]["x"], list)
```

Comment:
Also update the doc string of get_beamlet_index() then accordingly. There it still says, keys are x-direction and y-direction.